app/views/support_interface/application_forms/courses/edit.html.erb

```diff
@@ -32,6 +32,7 @@
         <%= f.govuk_radio_button :study_mode, :part_time, label: { text: 'Part time' } %>
       <% end %>
       <%= f.govuk_text_field :site_code, label: { text: 'Site code', size: 's' }, width: 5 %>
+      <%= f.govuk_text_field :recruitment_cycle_year, label: { text: 'Recruitment cycle year', size: 's' }, width: 5 %>
```

Comment:
Would the main use case for this be for changing it in the current cycle and the previous cycles? If so, how do you feel about using a couple of radios, similar to study mode? 

It feels a bit wrong to leave it so open with a text field when we're mainly going to be entering 2025 and 2024 in there.

Reply:
Good shout - I'll ask the Support team to see which they'd prefer 👌

Reply:
Also worth noting that myself and @PeteWilliams caught up to discuss the needs of this page as well as the current design. This is something we'll look at in future 👌